### DISCUSSION TITLE
Add setting to change the default position of the summary in the data viewer to the right side.

### DISCUSSION THREAD

**Discussion by Dakotah-Engel** [posit-dev/positron#4718] [Feedback]
By default the summary opens on the left side of the pane. I don't see an option to default this to the right side.
![Screenshot 2024-09-18 083536](https://github.com/user-attachments/assets/17f6424f-902b-46a1-af06-8e304537b067)
Preferred: 
![Screenshot 2024-09-18 083559](https://github.com/user-attachments/assets/520e792b-eb2f-46df-a986-3bf7d04fc99c)

**Comment by petetronic** [comment]
@Dakotah-Engel I'll turn this into an enhancement request for you, thank you for trying out Positron!
